src/azure-cli/azure/cli/command_modules/cdn/tests/latest/test_endpoint_scenarios.py

```diff
@@ -122,3 +122,88 @@ def _create_profile(self, resource_group, profile_sku):
         check = JMESPathCheck('sku.name', profile_sku)
         self.profile_create_cmd(resource_group, profile_name, sku=profile_sku, checks=check)
         return profile_name
+
+    @ResourceGroupPreparer()
+    def test_rulesEngine_crud(self, resource_group):
```

Comment:
please rename it to `test_rule_engine_crud`

Reply:
fixed